.gitignore

```diff
@@ -3,3 +3,5 @@
 *.swp
 *.swo
 __pycache__
+/.direnv
+/.envrc
```

Comment:
Would you mind splitting this off to a different commit?

Reply:
Done.